Configuration of trace sampling for RUM backend traces (#14577)
DataDog/documentation: content/en/real_user_monitoring/connect_rum_and_traces/_index.md

## How are APM quotas affected?

> Connecting RUM and Traces might increase significantly the APM ingested volumes. Use the initialization parameter `tracingSampleRate` to only keep a share of the backend traces starting from browserand mobile requests.

Reviewer: ```suggestion
Connecting RUM and traces may significantly increase the APM ingested volumes. Use the initialization parameter `tracingSampleRate` to keep a share of the backend traces starting from browser and mobile requests.
```
